infrastructure/terraform/components/sandbox/module_kms.tf

```diff
@@ -0,0 +1,46 @@
+module "kms" {
+  source = "git::https://github.com/NHSDigital/nhs-notify-shared-modules.git//infrastructure/modules/kms?ref=v1.0.8"
+
+  aws_account_id = var.aws_account_id
+  component      = var.component
+  environment    = var.environment
+  project        = var.project
+  region         = var.region
+
+  name                 = "main"
+  deletion_window      = var.kms_deletion_window
+  alias                = "alias/${local.csi}"
+  key_policy_documents = [data.aws_iam_policy_document.kms.json]
+  iam_delegation       = true
+}
+
+data "aws_iam_policy_document" "kms" {
```

Comment:
Don't think this is needed on the sandbox at the moment - the one in the API component is used for more stuff